### PR TITLE
Mention that downloading kubectl may need curl

### PR DESCRIPTION
**Pull request by anilkumar56**
Improvement to the kubectl doc.
fixes https://github.com/kubernetes/website/issues/32424

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Please sign CLA
More info
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by Anilkumar56** [comment]
I signed it

**Comment by Anilkumar56** [comment]
"I signed it"

**Comment by divya-mohan0209** [comment]
@Anilkumar56 please sign the CLA appropriately for us to review the PR further.

**Comment by Anilkumar56** [comment]
@mrbobbytables I have signed the EasyCLA as corporate employee but still its not passing on my PR. The github email with which i contributed matches the email i signed the CLA. Can you please take a look?

**Comment by divya-mohan0209** [comment]
Hey @Anilkumar56 , when you're pushing your PR have you set the username and user email as the ones that you have used to sign the CLA?

Please refer to [this](https://git-scm.com/docs/git-commit) link to configure them as variables for your environment before pushing the commit.

**Comment by Anilkumar56** [comment]
@divya-mohan0209  Yes, i have set the username and user email as the ones that i have used to sign the CLA.
still i am facing the EasyCLA  Failure.

**Comment by gmannos** [comment]
We are checking at our organization side if any step is missed during the CCLA signing. Please hold until then.

File: content/en/docs/tasks/tools/install-kubectl-linux.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ The following methods exist for installing kubectl on Linux:
 
 ### Install kubectl binary with curl on Linux
 
+## {{% heading "prerequisites" %}}
+
+`curl` tool is a prerequisites for installing kubectl binary with curl on Linux method.
```
Comment (zacharysarah) [inline content/en/docs/tasks/tools/install-kubectl-linux.md:30]:
```suggestion
If your Linux distribution doesn't have `curl` installed, you'll need to install `curl` before you can install the kubectl binary. For example:
    ```sh
    apt update
    apt-get install curl
    ```
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@anilkumar56 With the CLA signed, it's possible to review the PR itself.

To other reviewers: I'm genuinely not sure whether we should approve this PR. If other reviewers think we should approve it, then I'm open to the idea, with some changes.

@anilkumar56 I'm willing to bet you opened this PR because this is an undocumented requirement that you encountered, and I want to honor that experience. I'm willing to bet that other people encounter the same difficulty. 

That said--and it's rare for me to say this--this information may be overly specific at the level of a Kubernetes installation. What do other docs reviewers think?

**Comment by sftim** [comment]
I'm happy to list `curl` as an explicit prerequisite, but not tell the reader  specificallyhow to get `curl` installed. That way, they can use a search engine such as Google and search “How do I install curl on MyOperatingSystem” - and should find a clear answer.

The PR as is stands doesn't follow our website conventions for listing prerequisites. These should come before the text.

In the case of `curl`, we should make it clear that you need that tool only for a manual download, and that there are alternative tools that are equally valid. No need to _name_ any alternatives; again, Google searches for “alternative to curl tool” are likely to help if necessary.

**Comment by tengqm** [comment]
I'm a little bit concerned at naming 'curl' a prerequisite. You use, for example, 'wget' instead.

**Comment by sftim** [comment]
Some similar advice about prerequisites:
- https://kubernetes.io/docs/tasks/administer-cluster/verify-signed-images/#before-you-begin

The kubectl equivalent page states (semi-rendered):
>    Or if you have `curl` installed, use this command:
>   ```powershell
>    curl -LO "https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe"
>   ```

We could either that kind of wording.

**Comment by shannonxtreme** [comment]
> I'm a little bit concerned at naming 'curl' a prerequisite. You use, for example, 'wget' instead.

Maybe something like:

"Ensure that you have a network data transfer tool like `curl` installed"

**Comment by sftim** [comment]
> Ensure that you have a network data transfer tool like `curl` installed

Sort of, but if someone has `wget` installed then we also need to explain, especially for more novice users, to remember to adapt the instructions accordingly. Especially if our aims here include cutting down on the number of docs issues filed by folks whose only contact with the Kubernetes docs so far has been to find the page about setting up `kubectl` - that audience may well not have the level of tech familiarity that other pages can usually assume.

**Comment by sftim** [comment]
/retitle Mention that downloading kubectl may need curl

**Comment by reylejano** [comment]
@anilkumar56 , what do you think of @zacharysarah [suggestion](https://github.com/kubernetes/website/pull/32569#discussion_r896364836)?

**Comment by divya-mohan0209** [comment]
@anilkumar56 : Please advise if you'd be able to continue with this PR?

**Comment by divya-mohan0209** [comment]
@anilkumar56 I'll be closing off this PR for now due to no response from your side since April. Please feel free to reopen this whenever you are ready to address the comments & work upon it.
/close

**Pull request closed without merging**